tasks/queue-granules/index.js

```diff
@@ -63,15 +97,19 @@ async function queueGranules(event) {
         additionalCustomMeta: event.config.childWorkflowMeta,
       });
       if (executionArn) {
-        const queuedGranule = {
-          granuleId: granule.granuleId,
-          status: 'queued',
-          retries: 3,
-        };
-        await granulesApi.updateGranule({
-          prefix: event.config.stackName,
-          body: queuedGranule,
-        });
+        await Promise.all(granuleBatch.map((queuedGranule) =>
```

Comment:
We should use `p-map` here with some limited concurrency to be defensive against large batch sizes and overwhelming the Lambda API